interview-questions.md

```diff
@@ -8,44 +8,45 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a class?
 
-Your answer:
+Your answer: When calling a super, we ask that our new thing we're making, have an similiar outline as a standard feature.
```

Comment:
A lot of your interview answers could be expanded on further.  Consider talking about real-life experiences with these concepts and/or provide some examples.